src/wovnio/wovnphp/custom_domain/CustomDomainLangUrlHandler.php

```diff
@@ -7,13 +7,12 @@ public static function addCustomDomainLangToAbsoluteUrl($absoluteUrl, $targetlan
     {
         $currentCustomDomain = $customDomainLangs->getCustomDomainLangByUrl($absoluteUrl);
         $newLangCustomDomain = $customDomainLangs->getCustomDomainLangByLang($targetlang);
-        $changedUrl = self::changeToNewCustomDomainLang($absoluteUrl, $currentCustomDomain, $newLangCustomDomain);
-        return $changedUrl;
+        return self::changeToNewCustomDomainLang($absoluteUrl, $currentCustomDomain, $newLangCustomDomain);
     }
 
     public static function changeToNewCustomDomainLang($absoluteUrl, $currentCustomDomain, $newLangCustomDomain)
     {
-        if (!empty($currentCustomDomain) && !empty($newLangCustomDomain) && $currentCustomDomain->getLang() !== $newLangCustomDomain->getLang()) {
+        if (!empty($currentCustomDomain) && !empty($newLangCustomDomain)) {
```

Comment:
what's the significance of removing this check? If both were the same language, it seems useless to continue.

Reply:
oh yeah, this is indeed a new scenario.